### ISSUE TITLE
Chai `lengthOf` matcher not compatible with ethers/waffle objects

### ISSUE THREAD

**Issue by ItsNickBarry** [EthWorks/Waffle#420]
```
let quorum = ethers.constants.One;
let signers = await ethers.getSigners();

expect(quorum).to.be.at.least(1);

expect(signers.length).to.be.at.least(quorum);
expect(signers).to.have.lengthOf.at.least(1);
expect([1]).to.have.lengthOf.at.least(quorum);
expect(signers).to.have.lengthOf.at.least(quorum);
```

All statements run successfully except for the last one.

The error message is not particularly helpful: `AssertionError: Expected "[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]" to be at least 1`

Previously reported to [Hardhat](https://github.com/nomiclabs/hardhat/issues/1157).

**Comment by vanruch** [comment]
Thanks for reporting @ItsNickBarry. Until there's fix for this, you can cast expected length to normal number
`expect(signers).to.have.lengthOf.at.least(quorum.toNumber())`

**Comment by marekkirejczyk** [comment]
Released in [3.2.2](https://github.com/EthWorks/Waffle/releases/tag/3.2.2)
